apps/passport-client/package.json

```diff
@@ -33,6 +33,7 @@
     "json-bigint": "^1.0.0",
     "jubjub-signature-pcd": "0.0.5",
     "pako": "^2.1.0",
+    "pcd-country-identity": "0.0.5",
```

Comment:
[npm](https://www.npmjs.com/package/pcd-country-identity) says this is deprecated - looks like we should move to [anon-aadhaar-pcd](https://github.com/privacy-scaling-explorations/anon-aadhaar/tree/main/packages/anon-aadhaar-pcd)?

Reply:
Hey @rrrliu,
Thanks for your review. 
And yes we updated a lot of things since this PR, I'll update it.